frontend/src/components/Search/SearchForm/AiSearch/index.tsx

```diff
@@ -75,8 +82,19 @@ export const AiSearch: React.FC<any> = ({}) => {
 				dateSuggestion.endDate,
 				dateSuggestion.selectedPreset,
 			)
-			setAiMode(false)
-			setAiQuery('')
+			exitAiMode()
+		}
+	}
+
+	const setSubmittedQuery = () => {
+		if (aiSuggestion) {
+			setQuery(aiSuggestion.query)
+			setPendingDates(
+				dateSuggestion.startDate,
+				dateSuggestion.endDate,
+				dateSuggestion.selectedPreset,
+			)
+			exitAiMode()
```

Comment:
When I submitted the query it cleared the filters on me. Editing worked fine.

https://www.loom.com/share/53be6dbfd8b94dbd8709b621b3d50756?from_recorder=1&focus_title=1